src/screens/Assistant/SelectFavouritesBottomSheet.tsx

```diff
@@ -0,0 +1,249 @@
+        (departure) => departure.id === favouriteId,
+      );
+      if (!favourite) return;
+      addFrontPageFavouriteDeparture(favourite);
+    } else if (favouriteInFavourites) {
+      removeFrontPageFavouriteDeparture(favouriteInFrontpageFavourites);
+    }
+  };
+
+  return (
+    <BottomSheetContainer>
+      <ScreenHeaderWithoutNavigation
+        title={t(SelectFavouriteDeparturesText.header.text)}
+        leftButton={{
+          type: 'cancel',
+          onPress: close,
+          text: 'Close',
+        }}
+        color="background_1"
+      />
+
+      <ScrollView style={styles.flatListArea}>
+        {favoriteDepartures.length > 0 && (
+          <>
+            <ThemeText style={styles.questionText} type="heading__component">
+              {t(SelectFavouriteDeparturesText.title.text)}
+            </ThemeText>
+
+            <View>
+              {favouriteItems &&
+                favouriteItems.map((departureDetails, index) => {
+                  return (
+                    <View key={departureDetails.id}>
+                      <SelectableFavouriteDeparture
+                        handleSwitchFlip={handleSwitchFlip}
+                        favouriteId={departureDetails.id}
+                        active={
+                          frontPageFavouriteDepartures.find(
+                            (departure) => departure.id === departureDetails.id,
+                          )
+                            ? true
+                            : false
+                        }
+                        departureStation={departureDetails.quayName}
+                        lineIdentifier={departureDetails.lineLineNumber ?? ''}
+                        lineName={
+                          departureDetails.lineName ??
+                          t(
+                            SelectFavouriteDeparturesText.departures
+                              .allVariations,
+                          )
+                        }
+                        lineTransportationMode={
+                          departureDetails.lineTransportationMode ?? LegMode.BUS
+                        }
+                      />
+                      {favouriteItems.length - 1 !== index && (
+                        <SectionSeparator />
+                      )}
+                    </View>
+                  );
+                })}
+            </View>
+          </>
+        )}
+        {!favouriteItems.length && (
+          <MessageBox type="info">
+            <ThemeText>
+              {t(SelectFavouriteDeparturesText.noFavourites.text)}
+            </ThemeText>
+          </MessageBox>
+        )}
+      </ScrollView>
+
+      <FullScreenFooter>
+        <Button
+          interactiveColor="interactive_0"
+          text={t(SelectFavouriteDeparturesText.confirm_button.text)}
+          accessibilityHint={t(
+            SelectFavouriteDeparturesText.confirm_button.a11yhint,
+          )}
+          onPress={() => {
```

Comment:
can be simplified to  `onPress={close}` :)

Reply:
Ah, that's true! Fixing, thanks :)